docs/Basic-Guide.md

```diff
@@ -141,7 +141,6 @@ INFO:mlagents.learn:{'--curriculum': 'None',
  '--run-id': 'first-run',
  '--save-freq': '50000',
  '--seed': '-1',
- '--slow': False,
```

Comment:
Actually, none of this is accurate anymore. It looks like
```
INFO:mlagents.trainers:CommandLineOptions(debug=False, num_runs=1, seed=-1, env_path=None, run_id='first-run', load_model=False, train_model=True, save_freq=50000, keep_checkpoints=5, base_port=5005, num_envs=1, curriculum_folder=None, lesson=0, slow=False, no_graphics=False, multi_gpu=False, trainer_config_path='config/trainer_config.yaml', sampler_file_path=None, docker_target_name=None, env_args=None, cpu=False)
INFO:mlagents.envs:Start training by pressing the Play button in the Unity Editor.
```
today, Maybe just delete this part?

Reply:
I agree to delete it. It does not add anything.